x/capability/sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+sonar.organization=cosmos
+
+sonar.projectName=Cosmos SDK - x/capability
```

Comment:
@julienrbrt can you create this project on sonarcloud ?

Reply:
Created at: https://sonarcloud.io/project/overview?id=cosmos-sdk-x-capability